server/app/middlewares/users.js

```diff
@@ -0,0 +1,20 @@
+import createUser from './../controllers/users';
```

Comment:
I think this logic should be done in the controller and not in a middle-ware. Please share the reason why you have decided to do it this way

Reply:
I learned it from my LFA in the last boot camp, please what better way can it be done?